conductor/src/main.rs

```diff
@@ -85,6 +85,10 @@ async fn run(metrics: CustomMetrics) -> Result<(), ConductorError> {
         .unwrap_or_else(|_| "".to_owned())
         .parse()
         .expect("error parsing GCP_PROJECT_NUMBER");
+    let is_loadbalancer_public: bool = env::var("IS_LOADBALANCER_PUBLIC")
+        .unwrap_or_else(|_| "false".to_owned())
```

Comment:
Only a question.  We talked about making the default `true` here.  Having it `false` is fine just wanted to make sure that is what you intended.

Reply:
Agreed. Let's make it true by default.

Reply:
I would also change operator to have public true by default